packages/block-library/src/audio/index.js

```diff
@@ -23,6 +23,7 @@ export const settings = {
 		attributes: {
 			src: 'https://upload.wikimedia.org/wikipedia/commons/d/dd/Armstrong_Small_Step.ogg',
 		},
+		viewportWidth: 350,
```

Comment:
Could you explain a little why all of these changes to the `example` in `packages/block-library` are necessary? Not sure I understand.

Reply:
The `BlockPreview` component scales the content with given viewportWidth to fit large blocks relative to given container.
I agree, these numbers are confusing. I introduced much simpler scaling approach (ex: 0.75) [here](https://github.com/WordPress/gutenberg/pull/45719/files#diff-ba97676c79b4678b29dcdb77d3812f7b160b824df934c1289994831df2ce0c0fR65) which is much simpler.

This is required for blocks such as `audio`, to fit within the container.

Reply:
Aha, I see. Is this a new API?

Reply:
It's an existing API. Here is the ref from [the docs](https://github.com/WordPress/gutenberg/pull/45719/files#diff-a2de963e686384ef0f566d26844f206f08969e704113bd4a9f45151ff629c567R219)